External/FEXCore/include/FEXCore/Debug/InternalThreadState.h

```diff
@@ -118,6 +127,14 @@ namespace FEXCore::Core {
     std::shared_mutex ObjectCacheRefCounter{};
     bool DestroyedByParent{false};  // Should the parent destroy this thread, or it destory itself
 
+    void *operator new(size_t size) {
```

Comment:
Leftovers

Reply:
Required due to raw new usage. Added a comment.